saleor/graphql/translations/mutations/utils.py

```diff
@@ -475,10 +477,24 @@ def get_results(cls, instances_data_with_errors_list, reject_everything=False):
     def post_save_actions(cls, info, created_translations, updated_translations):
         manager = get_plugin_manager_promise(info.context).get()
 
+        translation_created_webhooks = get_webhooks_for_event(
```

Comment:
Maybe worth to wrap these actions in `if` condition? This would solve us the DB queries in case of not having anything to trigger?
```python
if created_translations:
    translation_created_webhooks = get_webhooks_for_event(
    ...
    for translation in created_translations:
if updated_translations:
    translation_updated_webhooks = get_webhooks_for_event(
    ...
    for translation in updated_translations:
```